PennMobile/New Group/AccountPageViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  AccountPageViewController.swift
+//  PennMobile
+//
+//  Created by Andrew Antenberg on 9/26/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AccountPageViewController: UIViewController, ShowsAlertForError, UITableViewDelegate, UITableViewDataSource {
+    var account: Account!
+    let tableView = UITableView(frame: .zero, style: .insetGrouped)
+    
+    var profileInfo = [(text: "Name", info: " "), (text: "Username", info: " "), (text: "Email", info: " ")]
+    var educationInfo = [(text: "Graduation Term", info: " "), (text: "School", info: " "), (text: "Major", info: " ")]
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupView()
+        setupTableView()
+        guard Account.isLoggedIn else {
+            self.showAlert(withMsg: "Please login to use this feature", title: "Login Error", completion: { self.navigationController?.popViewController(animated: true)} )
+            return
+        }
```

Comment:
You can use if let, and set a root level Account object to avoid the unwrapping in subsequent calls.